etna/core/models/settings.py

```diff
@@ -12,7 +13,35 @@
 
 @register_setting(icon="list-ul")
 class SiteSettings(BaseSetting, ClusterableModel):
+    beta_banner_standfirst = models.CharField(
+        max_length=200,
+        default="What does the 'beta' label mean?",
+        verbose_name="standfirst",
+    )
+    beta_banner_link = models.URLField(blank=True, verbose_name="link to")
```

Comment:
@mozaki1 If we need to accommodate bot internal (without scheme/domain) or external (with scheme/domain) here, we'll want to go with `models.CharField` instead. `models.URLField()` fails validation when the url doesn't include the scheme/domain.

Reply:
Hi Andy, based on  https://docs.djangoproject.com/en/4.0/ref/forms/widgets/#django.forms.URLInput

Summary below :

1. Normalizes to: A string.
2. Uses [URLValidator] to validate that the given value is a valid URL.
3. Error message keys: required, invalid
4. Has three optional arguments max_length, min_length, and empty_value which work just as they do for [CharField]

Number 2 may be relevant, which is not available in CharField , and since we know this is a URL , maybe this is a better option? after all this is what is recommended.

Reply:
@mozaki1 try sticking in `/about-us/` and see what happens :)